src/snowflake/snowpark/_internal/code_generation.py

```diff
@@ -351,11 +349,13 @@ def get_lambda_code_text(code_text: str) -> str:
             # session.udf.register(
             #    lambda x, y: x + y, ...
             # )
+            # code_text in this case is "lambda x, y: x + y, ..."
             code_text = f"wrapper({code_text})"
         elif "unmatched ')'" in str(exc):
             # handle case like:
             # session.udf.register(
             #    lambda x, y: x + y, ...)
+            # code_text in this case is "lambda x, y: x + y, ...)"
```

Comment:
This final closing parenthesis is very interesting :)

Reply:
yes, I was surprised when seeing this -- "Python what are u doing here?!"